data/studentApplyData.js

```diff
@@ -127,9 +126,10 @@ const studentProcess = [
   {
     title: (
       <>
-        <a href="https://www.facebook.com/events/783414215343789">
+        {/* <a href="https://www.facebook.com/events/783414215343789">
           Info Session
-        </a>
+        </a> */}
```

Comment:
Comments are temporary here until we have a facebook event

Reply:
should probably make the facebook link an additional variable, but can defer for later